src/main/java/racingcar/Racingcar.java

```diff
@@ -0,0 +1,79 @@
+package racingcar;
+
+import racingcar.ui.InputView;
+import racingcar.ui.ResultView;
+
+import java.util.*;
+import java.util.stream.IntStream;
+
+public class Racingcar {
+    private static final int RANDOM_VALUE = 10;
+    private static final int MOVE_CONDITION = 3;
+    private static final int MAX_INPUT_VALUE = 10000;
+    private static final Random random = new Random();
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        int numberOfCar = inputNumberOfCar(scanner);
+        int numberOfAttempt = inputNumberOfAttempt(scanner);
+
+        if (!isMaximumInput(numberOfCar, numberOfAttempt)) {
+            gameLogic(numberOfCar, numberOfAttempt);
+        }
+    }
+
+    public static boolean isMaximumInput(int numberOfCar, int numberOfAttempt) {
+        if(numberOfCar > MAX_INPUT_VALUE && numberOfAttempt > MAX_INPUT_VALUE)
+            return true;
+        return false;
+    }
+
+    public static void gameLogic(int numberOfCar, int numberOfAttempt) {
+        List<Car> carList = initCarList(numberOfCar);
```

Comment:
```suggestion
        List<Car> cars = initCars(numberOfCar);
```
자료구조에 의존적인 변수명은 좋지 않습니다.
List가 아닌 Set 으로 변경되었을 때 변수 이름도 같이 바뀌어야 할거에요
이런 이유로 복수형으로 표현할 수 있다면 복수형을 사용하는 것이 좋습니다.